cpu/esp8266/periph/i2c.c

```diff
@@ -130,7 +130,7 @@ static int _i2c_read_byte (_i2c_bus_t* bus, uint8_t* byte, bool ack);
 static int _i2c_arbitration_lost (_i2c_bus_t* bus, const char* func);
 
 /* implementation of i2c interface */
-
+#include "xtimer.h"
```

Comment:
Why add xtimer?

Reply:
@MrKevinWeiss There is no reason. Probably, I forgot to to remove it when I tried to do some timings using xtimer. Should I remove it?

Reply:
Yes please!